app/src/main/java/info/nightscout/androidaps/plugins/general/themes/ThemeSwitcherPlugin.kt

```diff
@@ -40,20 +40,20 @@ class ThemeSwitcherPlugin @Inject constructor(
 
     override fun onStart() {
         compositeDisposable.add(rxBusWrapper.toObservable(EventPreferenceChange::class.java).subscribe {
-            if (it.isChanged(resourceHelper, id = R.string.key_use_darkmode)) switchTheme()
+            if (it.isChanged(resourceHelper, id = R.string.key_useDarkmode)) switchTheme()
```

Comment:
i have to change name because the old var was boolean and the system crashed to your changes

Reply:
A fresh build should care not what the old var was. This is a rename to be consistent with the rest of the code.

Reply:
whatever. With your var name the system crashed and i had to rename it. this is because the boolean value is stored persistent and included in the export of the settings.

Reply:
Ah, that explains it. I never exported my settings in between builds. Renaming it back should then fix it again, since it should be "forgotten" with the new build. But, no big deal. Glad it works for you now.